test/extended/templates/templateservicebroker_security.go

```diff
@@ -68,7 +69,15 @@ var _ = g.Describe("[templates] templateservicebroker security test", func() {
 		brokercli = client.NewClient(&http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}, adminClientConfig.Host+templateapi.ServiceBrokerRoot)
 
 		viewuser = createUser(cli, "viewuser", bootstrappolicy.ViewRoleName)
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "view", viewuser.Name, "-n", "openshift").Execute()
```

Comment:
attempted to clarify need in comments here and in comments in provision.go wrt the SAR on getting templates that is now in